### DISCUSSION TITLE
Furo should switch to dark theme if `@media(prefers-color-scheme: dark)`

### DISCUSSION THREAD

**Discussion by jrief** [pradyunsg/furo#844] [Potential Issue] [answered]
A first time visitor of a Furo site always ends up in the light theme. Furo only queries the media type `prefers-color-scheme` if the user [clicks on the provided button](https://github.com/pradyunsg/furo/blob/fd3e7b94e9959870aa3c8efb221de66be923421d/src/furo/assets/scripts/furo.js#L89). For users having set their browser into dark mode by default, the Furo theme should start automatically in dark mode.

This btw. would be possibly by adding one line of JS code to `furo.js`. Shall I create a pull request?

**Comment by dgw** [comment, marked as answer]
> A first time visitor of a Furo site always ends up in the light theme. Furo only queries the media type `prefers-color-scheme` if the user [clicks on the provided button](https://github.com/pradyunsg/furo/blob/fd3e7b94e9959870aa3c8efb221de66be923421d/src/furo/assets/scripts/furo.js#L89).

No, I don't think that's true. It sets a theme mode data attribute at load, defaulting to "auto":

https://github.com/pradyunsg/furo/blob/fd3e7b94e9959870aa3c8efb221de66be923421d/src/furo/theme/furo/base.html#L84-L86

Media queries in the CSS take it from there.

I've run into some browser/OS combinations where this kind of thing doesn't work properly, but that isn't Furo's fault.
